bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/RefreshingOAuth2CredentialsInterceptor.java

```diff
@@ -381,4 +393,17 @@ protected RetryState getRetryState(BackOff backoff) throws IOException {
       return RetryState.Interrupted;
     }
   }
+
+  /**
+   * Call {@link CredentialFactory#clearHttpTransport()} and retrieve a new {@link OAuth2Credentials}.
+   */
+  @VisibleForTesting
+  void refreshCredentials() {
+    CredentialFactory.clearHttpTransport();
+    try {
+      this.credentials = (OAuth2Credentials) CredentialFactory.getCredentials(credentialOptions);
```

Comment:
We made a check at an earlier point, but it definitely doesn't hurt to do it again here.